src/lib/pay-request/typed_clients/services/ledger/client.ts

```diff
@@ -103,17 +104,48 @@ export default class Ledger extends Client {
       const accountIdParam = filters.account_id as number[]
       const refundFilters: any = {
         ...filters,
-        ...accountIdParam && accountIdParam.length && { account_id: accountIdParam.join(',') },
+        ...accountIdParam && accountIdParam.length && {account_id: accountIdParam.join(',')},
         transaction_type: TransactionType.Refund
       }
 
       return client._axios
-        .get('/v1/transaction', { params: refundFilters })
+        .get('/v1/transaction', {params: refundFilters})
         .then(response => client._unpackResponseData<SearchResponse<Refund>>(response));
     }
   }))(this)
 
   transactions = ((client: Ledger) => ({
+    retrieve(id: string): Promise<Transaction | undefined> {
+      return client._axios
+        .get(`/v1/transaction/${id}`, { params: {override_account_id_restriction: true}})
+        .then(response => client._unpackResponseData<Transaction>(response))
+        .catch(handleEntityNotFound('Transaction', id))
+    },
+
+    retrieveRelatedTransactions(id: string, params: RelatedTransactionsRequest): Promise<TransactionsForTransactionResponse | undefined> {
+      return client._axios
+        .get(`/v1/transaction/${id}/transaction`, {params})
+        .then(response => client._unpackResponseData<TransactionsForTransactionResponse>(response));
+    },
+
+    list(
+      filters: ListTransactionForAccountRequest | ListTransactionRequestWithAccountOverrideRequest
+    ): Promise<SearchResponse<Transaction> | undefined> {
+      const accountIdParam = filters.account_id as number[]
+      const paymentFilters: any = {
+        ...filters,
+
+        // @TODO(sfount): this is to work around querystring.stringy() not using a comma separated value
```

Comment:
👀 copying todos @sfount is probably not a good way to continue a pattern but I guess this can all be cleaned up and tackled at once with this being grepped